npm-scripts.js

```diff
@@ -211,3 +213,23 @@ function execute(command)
 		process.exit(1);
 	}
 }
+
+function installMsysMake()
+{
+	var res = spawnSync('where', [ 'python3.exe' ]);
+	if (res.status !== 0)
+	{
+		res = spawnSync('where', [ 'python.exe' ]);
+		if (res.status !== 0)
+		{
+			console.log('Cannot find Python executable.');
+			process.exit(1);
+		}
+	}
+	execute(String(res.stdout).trim() + ' worker\\scripts\\getmake.py');
+}
+
+function addMsysToPath()
+{
+	process.env['PATH'] = process.cwd() + '\\worker\\out\\msys\\bin;' + process.env['PATH'];
```

Comment:
`process.cwd()` is probably not a good idea, since it can be called from anywhere. `__dirname` should be a better alternative

Reply:
But the paths in all commands are relative to the cwd.

Reply:
Hm... yes, other paths are implicitly relative to cwd, but that isn't necessarily what they should be.

Reply:
Yet this is what they are. :slightly_smiling_face: Currently it looks like the script (with package.json) can be anywhere, but everything else should be in the current directory, rather than vice versa.